package.json

```diff
@@ -23,14 +26,8 @@
     "eslint-plugin-react": "7.16.0",
     "eslint-plugin-react-hooks": "2.2.0"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/pmedianetwork/eslint-config.git"
+  "devDependencies": {
+    "mocha": "7.1.0"
```

Comment:
can we use `jest` for testing like in all other repos?

Reply:
The wrapper is made for mocha, but let me check if I can hack it

Reply:
Documentation says `mocha` but replacing it for jest worked 😃 
https://github.com/pmedianetwork/eslint-config/pull/64/commits/cb8d029e9df365f3ca169cbe924dfa2567061a12

Reply:
awesome!